### PR TITLE
bug 1808170: Sync with upstream

### PR DESCRIPTION
**Pull request by ingvagabund**
Also
- dropping test directory and no longer used files
- replacing Makefile with the upstream one
- changing import path prefix to sigs.k8s.io/cluster-capacity
- bumping k8s version to 1.16

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,9 +1,7 @@
 approvers:
 - ingvagabund
-- aveshagarwal
-- hodovska
+- dameni
```
Comment (damemi) [inline OWNERS:3]: @damemi :)

File: OWNERS [hunk 2]
```diff
@@ -1,9 +1,7 @@
...
 reviewers:
 - ingvagabund
-- aveshagarwal
-- hodovska
+- dameni
```
Comment (damemi) [inline OWNERS:6]: @damemi

File: OWNERS (hunk 1 above)
Comment (ingvagabund) [reply OWNERS]: Sorry :) Fixed

**Pull request merged**